src/test/java/cat/udl/eps/softarch/demo/steps/RegisterBatchStepDefs.java

```diff
@@ -0,0 +1,20 @@
+package cat.udl.eps.softarch.demo.steps;
+
+import cat.udl.eps.softarch.demo.domain.Batch;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class RegisterBatchStepDefs {
+    public static Batch batch;
+
+    @When("^I register a new batch$")
+    public void registration(){
+        batch = new Batch();
```

Comment:
At least the @When should make use of the API being tested, something similar to what is done in https://github.com/UdL-EPS-SoftArch/mycircular-api/blob/main/src/test/java/cat/udl/eps/softarch/demo/steps/CreateTransDefs.java#L79